workbench-core/example/infrastructure/integration-tests/support/resources/datasets/dataset.ts

```diff
@@ -67,11 +67,15 @@ export default class Dataset extends Resource {
     return response;
   }
 
+  public async singlePartFileUploadUrl(body: { fileName: string }): Promise<AxiosResponse> {
```

Comment:
Since this is a method, should it be a verb? Maybe `generateSinglePartUploadUrl`?

Reply:
changed